src/natpmp.cpp

```diff
@@ -176,6 +176,21 @@ bool natpmp::should_log() const
 	return m_callback.should_log_portmap(aux::portmap_transport::natpmp);
 }
 
+void natpmp::mapping_log(char const* op, mapping_t const& m) const
+{
+	if (should_log())
+	{
+		log("%s-mapping: proto: %s port: %d local-port: %d action: %s ttl: %" PRId64
+			, op
+			, m.protocol == portmap_protocol::none
+			? "none" : to_string(m.protocol)
```

Comment:
"none" using only for #ifndef TORRENT_DISABLE_LOGGING
![default](https://cloud.githubusercontent.com/assets/920705/26523876/af054b4a-432a-11e7-94df-7611cc55f260.png)